src_c/_pygame.h

```diff
@@ -190,8 +190,9 @@ struct SDL_BlitMap {
 };
 #define SDL_COPY_RLE_DESIRED 0x00001000
 
-SDL_bool
-PG_SurfaceHasRLE(SDL_Surface *surface);
+#define PG_SurfaceHasRLE(surface) \
+    ((surface == NULL) ? 0 : (surface->map->info.flags & SDL_COPY_RLE_DESIRED))
```

Comment:
```suggestion
    (((surface) == NULL) ? 0 : ((surface)->map->info.flags & SDL_COPY_RLE_DESIRED))
```
Helps avoid some macro pitfalls.

If you want to further simplify the expression you can also do
```c
    ((surface) && ((surface)->map->info.flags & SDL_COPY_RLE_DESIRED))
```

Reply:
So the surface can be expression itself instead of a value? Okay.

I prefer the `==NULL` checks for explicitness. Like doing `is None` instead of a boolean check in Python.